packages/app-store/closecom/pages/setup/index.tsx

```diff
@@ -66,7 +66,8 @@ export default function CloseComSetup() {
                 form={form}
                 handleSubmit={async (values) => {
                   const { returnTo } = query;
-                  const url = new URL("/api/integrations/closecom/add");
+                  const baseUrl = window.location.origin;
+                  const url = new URL("/api/integrations/closecom/add", baseUrl);
```

Comment:
Hey, we have introduced a class called WebAppURL. Can you use it instead of window.location.origin?

Reply:
Is there an issue with `window.location.origin` technically speaking ?

Reply:
No issue, but when we provide the same here, why use the window object?

Reply:
No issue, we provide the similar thing.